WinCompatibilityPack/WinCompatibilityPack.psm1

```diff
@@ -870,11 +868,16 @@ C:\PS> Get-Module -ListAvailable
 function Add-WindowsPSModulePath
 {
 
-    if (-not $IsWindows)
+    if ($PSVersionTable.PSEdition -eq 'Core' -and -not $IsWindows)
```

Comment:
Do we need to worry about the version of PS on nano that reports `Core` but isn't really PowerShell Core?

Reply:
At this time, I wouldn't worry about Windows PowerShell Core 5